.gitignore

```diff
@@ -177,7 +177,6 @@ samples/client/petstore/python-tornado/.venv/
 # PHP
 samples/client/petstore/php/OpenAPIClient-php/composer.lock
 samples/openapi3/server/petstore/php-symfony/SymfonyBundle-php/composer.lock
-samples/openapi3/server/petstore/php-mezzio-ph/composer.lock
```

Comment:
@Articus Could you explain why do you consider to commit `composer.lock`? Since we build library not a project it's not really important to install exactly the same build on each contributor's machine. 

I don't get it, in 99% cases lock file is just an additional problem to me, especially with autodeploy. When I install new package and commit both `composer.json` and `composer.lock` then server runs `composer install` it always says like `Your lock file is different from deps, please run composer update bla-bla` So I need to manually SSH to remote server and run `composer update` over there which updates all deps and now `composer.lock` contains uncommitted changes, so next GitHub hook will overwrite lock file again and build will be broken, crazy. Maybe I miss something about composer CLI, cannot find a decent way to build remote server on a webhook.

@wing328 What do you think about committing `composer.lock` in our case?

Reply:
Fully agree about copmposer.lock :) But this line is removed because folder `samples/openapi3/server/petstore/php-mezzio-ph` does not exist any longer.

For historical reasons there were two samples for `php-mezzio-ph` - for OAS 2 spec file `samples/server/petstore/php-mezzio-ph` and for OAS 3 spec file `samples/openapi3/server/petstore/php-mezzio-ph`. During [my previous PR](https://github.com/OpenAPITools/openapi-generator/pull/8145) these folders became equal because according [this comment](https://github.com/OpenAPITools/openapi-generator/pull/8145#issuecomment-744177698) samples for OAS 2 spec are no longer needed. So I removed `samples/openapi3/server/petstore/php-mezzio-ph` entirely.